src/main/java/base/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package base;
+
+public interface AreaCalculable {
```

Comment:
The interface name `AreaCalculable` is not following the Java naming conventions for interfaces. Consider renaming it to something that represents an action or ability, such as `AreaCalculator`, which is more intuitive and aligns with common practice.